examples/pixels/package.json

```
@@ -14,9 +14,10 @@
     "@testing-library/user-event": "^14.1.1",
     "@types/jest": "^29.2.4",
     "craco-less": "^2.0.0",
+    "date-fns": "^2.29.3",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-scripts": "^5.0.0",
+    "react-scripts": "^4.0.3",
```

```suggestion
    "react-scripts": "^5.0.0",
```
We should use react-scripts@5